### PR TITLE
Add Breeze latest

### PR DESCRIPTION
**Pull request by Jamestown**
Create a cask for the Breeze application and use the latest version.

This application is free to use although it shows a pop-up that makes you wait a few seconds each time it is started. You can remove this pop-up by purchasing the application. I wasn't sure if the license should be :freemium or :gratis.

### CODE REVIEW

File: Casks/breeze.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  app 'Breeze.app'
+
+  uninstall :quit => 'com.autumnapps.direct.Breeze'
+
+  zap :delete => [
+                  '~/Library/Caches/com.autumnapps.direct.Breeze',
+                  '~/Library/Preferences/com.autumnapps.direct.Breeze.plist',
+                 ]
+end
```
Comment (vitorgalvao) [inline Casks/breeze.rb]: It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).

File: Casks/breeze.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+cask :v1 => 'breeze' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://autumnapps.com/downloads/Breeze.dmg'
+  name 'Breeze'
+  homepage 'https://autumnapps.com/breeze/'
+  license :freemium
```
Comment (vitorgalvao) [inline Casks/breeze.rb]: Is this a free app that works indefinitely but with limitations that can be removed by paying?

File: Casks/breeze.rb (hunk 1 above)
Comment (Jamestown) [inline Casks/breeze.rb]: I will add a newline.

File: Casks/breeze.rb (hunk 2 above)
Comment (Jamestown) [inline Casks/breeze.rb]: Essentially yes. Without paying for it, you have to wait several seconds each time the app is launched before you can use it. Paying for the app will remove this wait time, although I do not believe you gain an additional functionality in doing so.

File: Casks/breeze.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/breeze.rb]: Are you still going to update this?

Comment (Jamestown) [inline Casks/breeze.rb]: I was waiting to see if I needed to change the license value before making any changes.

File: Casks/breeze.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/breeze.rb]: Yes.

**Comment by vitorgalvao** [comment]
Fixed in https://github.com/vitorgalvao/homebrew-cask/commit/b6ccc4ac1c0c4839fc1cca04187776cca7d777f4.

**Pull request closed without merging**